NBitcoin/Crypto/Cryptsharp/Pbkdf2.cs

```diff
@@ -74,7 +74,7 @@ internal class Pbkdf2 : Stream
 		/// </param>
 		/// <param name="iterations">The number of iterations to apply.</param>
 #if USEBC || WINDOWS_UWP || NETCORE
-		public Pbkdf2(IMac hmacAlgorithm, byte[] salt, int iterations)
+		internal Pbkdf2(IMac hmacAlgorithm, byte[] salt, int iterations)

```

Comment:
Given the class is now public but its only constructor is internal, if i am in a netcore project then how can I create and new instance?